hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -261,7 +261,9 @@ module top_${top["name"]} #(
     .irq_timer_i          (intr_rv_timer_timer_expired_0_0),
     .irq_external_i       (irq_plic),
     .irq_fast_i           (15'b0),// PLIC handles all peripheral interrupts
-    .irq_nm_i             (1'b0),// TODO - add and connect alert responder
```

Comment:
I like the idea of having the conversion inside the `rv_core_ibex` wrapper. Related to that, I suggest to also remove `irq_fast_i` here and tie it off inside the wrapper. That's IMO the task of a wrapper and as the comment here says, we anyway don't use these.

Reply:
Sure, I'll tie that one off.